libs/blocks/section-metadata/section-metadata.js

```diff
@@ -15,13 +15,13 @@ function handleBackground(div, section) {
   }
 }
 
-export async function handleStyle(text, section) {
+export async function handleStyle(text, section, delay) {
```

Comment:
<img width="376" alt="Screen Shot 2024-02-05 at 12 58 42 PM" src="https://github.com/adobecom/milo/assets/2086146/67bc6460-0a14-41cf-b109-a8ad51ca9460">
Should we make this arg optional so the other files using this function don't need it. 
```suggestion
export async function handleStyle(text, section, delay = null) {
```

Reply:
Another thought, Rather than passing delay into `handleStyle()`, 
can we just get metadata.delay in the `handleStickySection()` below so we don't have to inject metadata.delay props into the metadata.style function?

Reply:
Incorporated the suggestion. Thanks for the review.